modules/op-signing-secrets-in-tekton-chains.adoc

```diff
@@ -15,4 +15,19 @@ Currently, {tekton-chains} supports the `x509` and `cosign` signature schemes.
 Use only one of the supported signature schemes.
 ====
 
-To use the `x509` signing scheme with {tekton-chains}, store the `x509.pem` private key of the `ed25519` or `ecdsa` type in the `signing-secrets` Kubernetes secret.
+To use the `x509` signing scheme with {tekton-chains}, you must fulfill the following requirements:
+
+* Store the private key in the `signing-secrets` with the `x509.pem` structure.
+* Store the private key as an unencrypted `PKCS #8` PEM file: `BEGIN PRIVATE KEY`.
+* The key is of `ed25519` or `ecdsa` type.
++
+[IMPORTANT]
+====
+Ensure that you store the `x509-pub.pem` public key in a safe place to verify later artifact attestations.
```

Comment:
What do you mean by a `safe place`? Public key can be stored anywhere, private key needs to be stored in a safe place.

Reply:
I understood this line from SME to be about not losing the public key. @jkhelil is that wrong? Maybe the wording should be something like: 
```
Ensure that you do not lose the `x509-pub.pem` public key for later artifact attestation verification.
```